src/adapter/mysql.cr

```diff
@@ -76,6 +76,32 @@ class Granite::Adapter::Mysql < Granite::Adapter::Base
     end
   end
 
+  def import(table_name : String, primary_name : String, primary_auto : Bool, fields, model_array)
+   params = [] of DB::Any
+    statement = String.build do |stmt|
+      stmt << "INSERT INTO #{quote(table_name)} ("
+      stmt << "#{quote(primary_name)}, " unless primary_auto
+      stmt << fields.map { |field| quote(field) }.join(", ")
+      stmt << ") VALUES "
+
+      model_array.each do |model|
+        next unless model.valid?
+        stmt << '('
+        stmt << "?," unless primary_auto
+        stmt << Array.new(fields.size, '?').join(',')
```

Comment:
The primary field references here can be removed once #169 gets merged, since fields will cover it by default.